CMakeLists.txt

```diff
@@ -214,74 +336,46 @@ configure_file(${PROJECT_SOURCE_DIR}/NEMS/src/ESMFConvenienceMacros.h
                ${PROJECT_BINARY_DIR}/NEMS/src/ESMFConvenienceMacros.h
                COPYONLY)
 
+target_link_libraries(ufs-weather-model-lib PUBLIC fv3atm )
+
 ###############################################################################
-### ufs-s2s-model
+### UFS executable
 ###############################################################################
-add_executable(
-    NEMS.exe
-
-    NEMS/src/MAIN_NEMS.F90
+add_executable(NEMS.exe
+               NEMS/src/MAIN_NEMS.F90
 )
-add_dependencies(NEMS.exe ufs-s2s-model-lib)
+add_dependencies(NEMS.exe ufs-weather-model-lib fms)
```

Comment:
something is wrong here.  the weather-model-lib depends on fms. and So there is no need for `fms` here.